src/customize-pop/gnome-tweaks-extensions/user-themes.md

```diff
@@ -0,0 +1,94 @@
+# Managing User Themes
+
+User themes include changes to window animations, desktop color schemes, window designs, icons, and workspaces. You can install custom GNOME Themes from opendesktop.org. The Pop!\_OS default themes can be re-applied using GNOME Tweaks or the by using Terminal commands.
+
+>**Note**: Applying custom GNOME theming options may cause system instability or glitchy behavior. Additionally, custom themes are not subject to testing by the System76 Quality Assurance team or application developers.
+
+## Enabling User Themes
+
+1. Launch the GNOME Extension Manager.
+
+    ![Launch GNOME Extension Manager](/images/user-themes/launch-ext-manager.png)
+
+2. Click `Browse` and search for `user themes`, then click `Install`.
+
+    ![Install User Themes](/images/user-themes/install-user-themes.png)
+
+3. Click `Installed` and ensure the toggle switch for `User Themes` is enabled.
+
+    ![Verify User Themes is Enabled](/images/user-themes/verify-enabled.png)
+
+## Adding a Custom User Theme
+
+> **Note**: Themes in Pop 22.04 must support GTK 4. Additionally, it may be necessary to disable [Pop!\_OS components](gnome-tweaks-extensions.md#built-in-extensions) from GNOME Extensions.
```

Comment:
I would say Pop!_OS instead of just Pop when you're referring to the OS version in formal documentation. Pop by itself is just the default theme's name.